frontend/src/Pages/officeManagerScreen.js

```diff
@@ -6,11 +6,21 @@ import K_Health_logo from "../Icons/logoK.svg";
 import React, { useState } from 'react';
 import leftArrow from "../Icons/leftArrow.svg";
 import rightArrow from "../Icons/rightArrow.svg";
+// import { userslist } from "./NamesList.js"
+
+import { useSelector } from "react-redux";
+import { selectDate } from "../redux/dateSlice.js";
+
 
 function OfficeManagerScreen() {
   const [date, setDate] = useState(new Date());
-  const weekday = ["Sunday","Monday","Tuesday","Wednesday","Thursday","Friday","Saturday"];
-  const month = ["January","February","March","April","May","June","July","August","September","October","November","December"];
+  const weekday = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+  const month = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
```

Comment:
```suggestion
  const WEEK_DAY = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
  const MONTH = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
```

and move it out of the function